sunkit_image/tests/conftest.py

```diff
@@ -95,6 +96,13 @@ def pytest_runtest_setup(item):
 def granule_map():
     return sunpy.map.Map(get_pkg_data_filename("dkist_photosphere.fits", package="sunkit_image.data.test"))
 
+@pytest.fixture()
+def granule_map_he():
+    granule_map = sunpy.map.Map(get_pkg_data_filename("dkist_photosphere.fits", package="sunkit_image.data.test"))
+    map = granule_map.data
+    map_norm = ((map - np.nanmin(map))/(np.nanmax(map) - np.nanmin(map))) # min-max normalization to [0, 1] 
```

Comment:
Is there no helper in scikit image to do the normalization to [0, 1]?

Reply:
Hmm... I am not seeing one. I could use an sklearn or scipy package instead.